pages/docs/guides/message-validation-kafka.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Message validation for Kafka
+description: This guide explains how you can validate AsyncAPI messages being sent via Kafka.
+weight: 140
+---
+
+## Introduction
+This guide explains all the ways you can validate AsyncAPI messages being sent via Kafka. It provides an overview of message validation in Kafka, focusing on the role of the Schema Registry. You will also be introduced to popular Schema Registry implementations such as Apicurio and Confluent Schema Registry.
```

Comment:
Let's remember not to start sentences with **antecedents.**

Reply:
Removed all antecedents, thank you!